examples/fluids/src/misc.c

```diff
@@ -150,32 +136,66 @@ PetscErrorCode RegressionTests_NS(AppCtx app_ctx, Vec Q) {
   PetscFunctionReturn(PETSC_SUCCESS);
 }
 
-// Get error for problems with exact solutions
-PetscErrorCode GetError_NS(CeedData ceed_data, DM dm, User user, Vec Q, PetscScalar final_time) {
-  PetscInt  loc_nodes;
-  Vec       Q_exact, Q_exact_loc;
-  PetscReal rel_error, norm_error, norm_exact;
+PetscErrorCode ComputeL2Error(Vec Q_loc, PetscReal l2_error[5], OperatorApplyContext op_error_ctx, CeedContextFieldLabel time_label,
+                              CeedScalar time) {
+  Vec       E;
+  PetscReal l2_norm[5];
+  PetscFunctionBeginUser;
+
+  if (time_label) CeedOperatorSetContextDouble(op_error_ctx->op, time_label, &time);
```

Comment:
Prefer using `UpdateContextLabel`, but that's currently not in `navierstokes.h`; feel free to add the signature there are use it here if you want.